agent/sshd/server.go

```diff
@@ -122,13 +123,21 @@ func (s *Server) sessionHandler(session sshserver.Session) {
 
 		os.Chown(pts.Name(), uid, -1)
 
+		remoteAddr := session.RemoteAddr().String()
+
 		logrus.WithFields(logrus.Fields{
 			"user":       session.User(),
 			"pty":        pts.Name(),
-			"remoteaddr": session.RemoteAddr().String(),
+			"remoteaddr": remoteAddr,
 			"localaddr":  session.LocalAddr().String(),
```

Comment:
please check if it really needs to use `.String()` here and on `remoteAddr` as I belive it will call it internally.

Reply:
I'll check but `.String()`  was in the original code,so I left it.

Reply:
I've confirmed that `.String()` isn't needed in the calls of logrus.WithFields, so I've deleted it.